docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
+
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
 
 ## Get started
 
-This is page will be dedicated to directing users on how to get started with Temporal.
+Ready to develop? [Install the Temporal server](install-temporal-server) and choose your SDK.
+
+→ [Go SDK](go-quick-start)
 
-This is placeholder content.
+→ [Java SDK](java-quick-start)
 
 ## Learn about Temporal
 
-etc...
+Building applications with Temporal is a paradigm shift. If you are new to Temporal, or workflow applications in general, get caught up on the [core concepts](concepts-overview) and common [use cases](use-cases-overview).
+
+Run the code samples to see how it works:
+
+→ [Go SDK samples](https://github.com/temporalio/go-samples)
+
+→ [Java SDK samples](https://github.com/temporalio/java-samples)
+
+## Best practices
+
+Want to master Temporal Workflows and recommended development techniques? Our guides will show you how.
+
+- [How to install Temporal](install-temporal-server)
```

Comment:
Wow didn't know links work without any path "./". I'm not suggesting this is broken it's a genuine observation.

Reply:
Chayup. https://v2.docusaurus.io/docs/markdown-features#referencing-other-documents

Reply:
Chayup: https://v2.docusaurus.io/docs/markdown-features#referencing-other-documents